api/prometheus.go

```diff
@@ -249,27 +389,209 @@ func (m Metrics) PrometheusMetric() []prometheus.Metric {
 func (wr WalletResponse) PrometheusMetric() []prometheus.Metric {
 	return []prometheus.Metric{
 		{
-			Name: "hostd_wallet",
+			Name: "hostd_wallet_scan_height",
 			Labels: map[string]any{
 				"address": wr.Address,
 			},
-			Value:     float64(wr.ScanHeight),
-			Timestamp: time.Now(),
+			Value: float64(wr.ScanHeight),
 		},
 		{
-			Name:      "hostd_wallet_spendable",
-			Value:     wr.Spendable.Siacoins(),
-			Timestamp: time.Now(),
+			Name: "hostd_wallet_spendable",
+			Labels: map[string]any{
+				"address": wr.Address,
+			},
+			Value: wr.Spendable.Siacoins(),
 		},
 		{
-			Name:      "hostd_wallet_confirmed",
-			Value:     wr.Confirmed.Siacoins(),
-			Timestamp: time.Now(),
+			Name: "hostd_wallet_confirmed",
+			Labels: map[string]any{
+				"address": wr.Address,
+			},
+			Value: wr.Confirmed.Siacoins(),
 		},
 		{
-			Name:      "hostd_wallet_unconfirmed",
-			Value:     wr.Unconfirmed.Siacoins(),
-			Timestamp: time.Now(),
+			Name: "hostd_wallet_unconfirmed",
+			Labels: map[string]any{
+				"address": wr.Address,
+			},
+			Value: wr.Unconfirmed.Siacoins(),
+		},
+	}
+}
+
+func (t TPoolResp) PrometheusMetric() (metrics []prometheus.Metric) {
+	return []prometheus.Metric{
+		{
+			Name:  "hostd_tpool_fee",
+			Value: t.RecommendedFee.Siacoins(),
 		},
 	}
 }
+
+func (a AccountResp) PrometheusMetric() (metrics []prometheus.Metric) {
+	for _, account := range a {
```

Comment:
Hosts can have millions of active accounts. I don't think this is a good idea.

Reply:
i agree this is not a good idea. removed from next commit